docs/source/udf/advanced.rst

```diff
@@ -0,0 +1,209 @@
+User-defined functions: advanced topics
+=======================================
+
+.. _tiled:
+
+Tiled processing
+----------------
+
+Motivation
+~~~~~~~~~~
+
+Many operations operations can be significantly optimized by working on stacks of frames.
+You can often perform `loop nest optimization <https://en.wikipedia.org/wiki/Loop_nest_optimization>`_
+to improve the `locality of reference <https://en.wikipedia.org/wiki/Locality_of_reference>`_,
+for example using `numba <https://numba.pydata.org/>`_, or using an optimized numpy function.
```

Comment:
Perhaps add this sentence to show relevance for our target audience?

Gain correction where a constant dark frame is subtracted from each input frame and the result is multiplied with a constant gain map is a typical application in electron microscopy where such optimizations are very effective.